charts/datadog/values.yaml

```
@@ -1305,6 +1305,10 @@ clusterAgent:
   #          username: datadog
   #          password: <YOUR_CHOSEN_PASSWORD>
 
+  ## clusterAgent.kubernetesApiserverCheck.disableUseComponentStatus -- Disable useComponentStatus for kubernetes_apiserver check
```

```suggestion
  # clusterAgent.kubernetesApiserverCheck.disableUseComponentStatus -- Set this to true to disable use_component_status for the kubernetes_apiserver check
```

A single `#` should populate the readme with the description